src/index.js

```diff
@@ -29,7 +30,10 @@ export default new Transformer({
   async transform({ asset, config }) {
     configure({ autoescape: false });
     let code = await asset.getCode();
-    let result = renderString(code, config);
+    let result = renderString(code, {
+      ...config,
+      file: basename(asset.filePath),
```

Comment:
I think `basename` or `filename` might be a better name for the variable than `file`. Otherwise lgtm.